requirements/common.in

```diff
@@ -15,10 +15,10 @@ Jinja2
 Markdown
 Pygments
 hyperlink
+jsx-lexer
 
 # Needed for manage.py
-# Upgrade blocked because ipython dropped Python 3.5 support.
-ipython==7.9.*
+ipython
```

Comment:
I guess they haven't come out with releases after 7.9.0, and that's why this is here?  OK.

Reply:
Actually, I didn't realize that they had released a new version and I had to manually trigger the update. I pushed a change for this in another of my PR that is open (#14734)